tests/ha/remove_node.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Remove a node both by its hostname and ip address
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    my $cluster_name = get_cluster_name;
+    my $node_01      = choose_node(1);
+    my $node_02      = choose_node(2);
+    my $node_01_ip   = get_ip($node_01);
+    my $node_02_ip   = get_ip($node_02);
+    my $remove_cmd   = 'crm cluster remove -y -c';
+    my $join_cmd     = 'crm cluster join -y -w /dev/watchdog -i eth0 -c';
```

Comment:
Maybe better to replace `eth0` here by `get_var('SUT_NETDEVICE', 'eth0')`?

(I just notice that there is also `eth0` hard-coded in the HAProxy test, maybe useful to change in a future PR?)